spec/hash-map.dd

```diff
@@ -392,13 +400,14 @@ $(H2 $(LNAME2 runtime_initialization, Runtime Initialization of Immutable AAs))
             aa = assumeUnique(temp);
         }
 
-        unittest
+        void main()
```

Comment:
do we not have a `SPEC_RUNNABLE_UNITTEST_EXAMPLE_RUN`?

Reply:
grepping for `UNIT` or `TEST` only found `DDOX_UNITTEST_HEADER`.